packages/react/menu/src/Menu.tsx

```diff
@@ -800,7 +801,7 @@ const MenuSubTrigger = React.forwardRef<MenuSubTriggerElement, MenuSubTriggerPro
     }, [pointerGraceTimerRef, onPointerGraceIntentChange]);
 
     return context.isSubmenu ? (
-      <MenuAnchor asChild {...props}>
+      <MenuAnchor asChild {...scope}>
```

Comment:
props were being spread here and on `MenuItemImpl` so we were composing the consumer click event twice.

Reply:
Yeah that's surprising 🤔